### PR TITLE
Use the trash icon

### PR DESCRIPTION
**Pull request by Ladsgroup**
Now we have the new wikit version, we can use it.

Bug: T268982

### CODE REVIEW

File: src/components/DeleteConditionButton.vue [hunk 1]
```diff
@@ -1,16 +1,18 @@
 <template>
 	<Button
+		iconOnly
 		class="delete-condition-button"
-		type="destructive"
-		variant="primary"
+		type="normal"
```
Comment (micgro42) [inline src/components/DeleteConditionButton.vue]: This should be `type="neutral"` not normal, sorry!

File: src/components/DeleteConditionButton.vue [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+		variant="quiet"
+		aria-label="Delete condition"
```
Comment (micgro42) [inline src/components/DeleteConditionButton.vue]: Could this be an i18n message?

**Review by micgro42 — CHANGES_REQUESTED** [review]

File: src/components/DeleteConditionButton.vue (hunk 2 above)
Comment (guergana) [reply src/components/DeleteConditionButton.vue]: +1

File: src/components/DeleteConditionButton.vue (hunk 1 above)
Comment (Ladsgroup) [reply src/components/DeleteConditionButton.vue]: Yeah, noticed it late :D

File: src/components/DeleteConditionButton.vue (hunk 2 above)
Comment (Ladsgroup) [reply src/components/DeleteConditionButton.vue]: Done

**Review by micgro42 — APPROVED** [review]

**Pull request merged**